### ISSUE TITLE
Contract Ops Questions

### ISSUE THREAD

**Issue by ef4** [cardstack/cardstack-token#42]
I see that the secure terminal needs to go through a software setup phase that is going to require network. But can we do all of that once *before* any private keys are generated, then disconnect from the network (and ideally never reconnect!), then generate private keys and do offline signing operations from then on?

Alternatively, is there a practical way to separate contract generation from signing? If so, could we just have multiple people run the generation steps independently and bring the resulting unsigned contracts into the clean room, use the secure terminal to verify that they are all identical (so nobody manipulated the contract or had malware), then use the secure terminal's private keys to sign the contract. The benefit of this is that much of the developer toolchain can stay off the secure terminal, reducing the attack surface significantly.

Basically I'm looking for ways to simplify the in-the-cleanroom operations to just "bring in a thing to be signed, get people to agree the thing is valid, sign the thing, and sneaker-net it out to be posted to the network".
